backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -950,7 +953,7 @@ const canEmailResend = (updatedAt: Date): boolean => {
   return !isTimeExpired(updatedAt, CONFIG.EMAIL_CODE_REQUEST_TIME)
 }
 
-const getTimeDurationObject = (time: number): { hours?: number; minutes: number } => {
+export const getTimeDurationObject = (time: number): { hours?: number; minutes: number } => {
```

Comment:
if you need t import this you should move it into a commonly used utils file (?) (if its just the test idk)

Reply:
I thought about it, but found this habit elsewhere …

Reply:
Done